app/templates/components/dialogs/prioritize-build-modal.hbs

```diff
@@ -0,0 +1,66 @@
+<Modal
+  class="repo-actions-modal"
+  @isVisible={{@isOpen}}
+  @onClose={{fn @onClose}}
+>
+  <div class="repo-actions-modal__top">
+    <h3 class="repo-actions-modal__header">
+      Prioritize your build
+    </h3>
+    <button
+      type="button"
+      title="Close modal"
+      class="pointer repo-actions-modal__close-button"
+      autofocus={{this.focusOnModal}}
+      {{on 'click' (fn @onClose)}}
+    >
+      <SvgImage @name="icon-failed" @class="icon" />
+    </button>
+  </div>
+  <div class="repo-actions-modal__body">
+    <p>
+      Bypass your scheduled build by fast tracking this one.
+    </p>
+    <p class="label-align">
+      SELECT AN OPTION
```

Comment:
I'd like to suggest managing text casing via css `text-transform: uppercase` property, as soon as it's a styling concern.

Reply:
Done.